packages/compass-export-to-language/src/modules/count-aggregation-stages-in-string.js

```diff
@@ -13,17 +13,33 @@ export function countAggregationStagesInString(str) {
   if (!dummySandbox) {
     dummySandbox = vm.createContext(Object.create(null), {
       codeGeneration: { strings: false, wasm: false },
-      microtaskMode: 'afterEvaluate'
+      microtaskMode: 'afterEvaluate',
     });
-    vm.runInContext([
```

Comment:
I just noticed afterwards that here we were already using vm.runInContext directly anyway.

Reply:
… to be fair, I only used this because it was quick and reliable. This is not a good long-term solution! :)

Reply:
Aaah now I remember what this was. OK so probably not releveant, then. Couldn't tell from the diff.